public/index.html

```diff
@@ -9,8 +9,22 @@
   <title>Ultimate Nike</title>
 </head>
 <body>
-  <div id="app"></div>
-  <div id="photo-gallery"></div>
-  <script src="bundle.js"></script>
-  <script src="http://localhost:3000/bundle.js"></script>
+  <div id="header"></div>
+  <div id="main">
+    <div id="left-column">
+      <div id="photo-module"></div>
+    </div>
+    <div id="right-column">
+      <div id="app"></div>
+    </div>
+  </div>
+    <!-- <div id="instagram"></div> -->
+  <div id="recommendations"></div>
+  <div id="footer"></div>
+  <script crossorigin src="http://localhost:3000/bundle-photo-module.js"></script>
```

Comment:
Please enlighten me on how `crossorigin` is used.  Thanks!

Reply:
I don't think it's having much practical effect here, I was just going by the React guidance here: https://reactjs.org/docs/cross-origin-errors.html